### PR TITLE
Corporate refactoring i guess

### PR DESCRIPTION
**Pull request by hrobohboy**
For this branch, I performed some *slight* refactoring of the current code structure. Anywhere that I saw applicable I removed the enums from the respective class and relocated them to an external Enums file. 
All enums can be found in Utilities/Enums. 

In order to reference these enums in pieces of code you just need to add the following code in the header:
`using RelhaxModpack.Utilities.Enums;`
If any enums need to be added into the future that are general enough to be exported, please create a new external enums file. *I'm looking at you Willster*

There were not any functionality changes in this so testing was done via the RelHaxUnitTests project.

### CODE REVIEW

**Comment by Willster419** [comment]
Very cool, thanks. I had made changes at the same time of this; are there any new enums that need to be moved to this style?

File: RelhaxModpack/RelhaxModpack/Database/DownloadInstructions.cs [hunk 1]
```diff
@@ -1,17 +1,7 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using RelhaxModpack.Utilities.Enums;
```
Comment (Willster419) [inline RelhaxModpack/RelhaxModpack/Database/DownloadInstructions.cs:1]: I see there was some cleaning of not used `using`s. I meant to do that. Eventually.

File: RelhaxModpack/RelhaxModpack/RelhaxModpack.csproj [hunk 2]
```diff
@@ -533,6 +555,9 @@
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
       <SubType>Designer</SubType>
     </EmbeddedResource>
+    <None Include="..\.editorconfig">
```
Comment (Willster419) [inline RelhaxModpack/RelhaxModpack/RelhaxModpack.csproj:558]: What's this?

**Review by Willster419 — COMMENTED** [review]
Left a comment about an extra file in the csproj

File: RelhaxModpack/RelhaxModpack/RelhaxModpack.csproj (hunk 2 above)
Comment (hrobohboy) [reply RelhaxModpack/RelhaxModpack/RelhaxModpack.csproj]: A file that vs added to "enforce coding standard in a project", it's been removed.

**Review by Willster419 — APPROVED** [review]

**Pull request merged**